test/v1alpha1/policy_groups_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2021 The Rode Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1_test
+
+import (
+	"context"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/rode/rode/proto/v1alpha1"
+)
+
+var _ = Describe("Policy Groups", func() {
+	var ctx context.Context
+
+	BeforeEach(func() {
+		ctx = context.Background()
+	})
+
+	Describe("Creating a policy group", func() {
+		When("the policy group name is valid", func() {
+			It("should create the policy group", func() {
+				expectedName := strings.ToLower(fake.LetterN(10))
+				_, err := rode.CreatePolicyGroup(ctx, &v1alpha1.PolicyGroup{
+					Name: expectedName,
+				})
+				Expect(err).NotTo(HaveOccurred())
+
+				group, err := rode.GetPolicyGroup(ctx, &v1alpha1.GetPolicyGroupRequest{Name: expectedName})
```

Comment:
nitpick: we could use the return value here as a way to assert that the policy group that was returned has the expected name

```suggestion
				createdPolicyGroup, err := rode.CreatePolicyGroup(ctx, &v1alpha1.PolicyGroup{
					Name: expectedName,
				})
				Expect(err).NotTo(HaveOccurred())

				group, err := rode.GetPolicyGroup(ctx, &v1alpha1.GetPolicyGroupRequest{Name: createdPolicyGroup.Name})
```